service/cmdcache/cmdcache.go

```diff
@@ -58,13 +73,13 @@ func (c *cmdCache) addCommand(cmd *clientpb.Command) {
 }
 
 // Get returns a batch of commands to propose.
```

Comment:
This comment doesn't appear correct, but it's the same on master.

Reply:
I see that the logic actually takes a batch of commands and marshals them into a string (hotstuff.Command)... It's a bit strange IMO.